src/pages/ServiceList/ServiceListPage.tsx

```diff
@@ -1,43 +1,20 @@
 import * as React from 'react';
-import { Alert } from 'patternfly-react';
 import ServiceListComponent from './ServiceListComponent';
+import * as MessageCenter from '../../utils/MessageCenter';
 
-type ServiceListState = {
-  alertVisible: boolean;
-  alertDetails: string;
-};
+type ServiceListState = {};
 
 type ServiceListProps = {
   // none yet
 };
 
 class ServiceListPage extends React.Component<ServiceListProps, ServiceListState> {
-  constructor(props: ServiceListProps) {
-    super(props);
-
-    this.state = {
-      alertVisible: false,
-      alertDetails: ''
-    };
-  }
-
   handleError = (error: string) => {
-    this.setState({ alertVisible: true, alertDetails: error });
-  };
-
-  dismissAlert = () => {
-    this.setState({ alertVisible: false });
+    MessageCenter.add(error);
   };
 
   render() {
     let alertsDiv = <div />;
```

Comment:
since alerts are gone, should we remove this div completely (or at minimum take it out and just put \<div/> directly in the return value?

Reply:
Missed that one, yep, we should remove that one.